.github/workflows/add_release_label.yml

```
@@ -0,0 +1,62 @@
+name: Add release label to PR and linked issues when PR gets merged
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types:
+      - closed
+
+jobs:
+  add_release_label:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 0 # This is needed to checkout all branches
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Install dependencies
+        run: yarn
```

Typically we use `yarn --immutable` in CI to ensure that we bail if the lockfile is stale. This ensures that we don't use unknown versions of packages, which might make a security audit difficult if we ever had reason to analyze what third-party packages were present for a particular run.

```suggestion
        run: yarn --immutable
```

---

This is now fixed with this [commit](https://github.com/MetaMask/metamask-extension/pull/19061/commits/d0c889cc8af9df156c8d57123dc0b91b1f1bd439).